packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -922,9 +921,12 @@ class ScrollableBaseNative extends Component {
 			this.childRef.didScroll(this.scrollLeft, this.scrollTop, dirHorizontal, dirVertical);
 		}
 		this.forwardScrollEvent('onScroll');
+		this.scrollStopJob.start();
 	}
 
-	scrollStopJob = new Job(this.scrollStopOnScroll.bind(this), scrollStopWaiting);
+	startScrollJob = new Job(this.scrollOnScroll, 16);
+
+	scrollStopJob = new Job(this.scrollStopOnScroll, scrollStopWaiting);
```

Comment:
Note: We don't need to call _bind_ for arrow functions in general, but we've experienced _this binding issue_ due to this line before. I'm not sure what was the cause. It could be an issue related to transpiling configuration. If it works well now, it's time to remove _bind_. :)

Reply:
Binding shouldn't be necessary because babel transpilation stores `this` in a closure. In the case of class members, the first transpilation step is to move it into the constructor and the second is to cache an ref to `this` and rewrite the usage of `this` in the function to the ref.

Here's an example from babel's REPL with a couple common variations of arrow function and class member transpilation to see it all in action.

http://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAEYEbGgbwFDWsMITQBQEpk10MB7AOwgBdoBbAUyoAtSATaAXj0I4D4iSJZgEsIAOgikGI8gHMA3MXQBfRSuIzZnaACZiDWnHoAnbQU79UgkeMnSmwuWujLixUsxPaAZgFdywFTCFNwCQg62UowOTsSqKMQQ9MAU7FzmfGHoNhJRms6urkA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=6.26.0&envVersion=